netplan_cli/cli/state.py

```diff
@@ -491,6 +504,46 @@ def query_resolved(cls) -> tuple:
             logging.debug('Cannot query resolved DNS data: {}'.format(str(e)))
         return (addresses, search)
 
+    @classmethod
+    def query_members(cls, ifname: str) -> List[str]:
```

Comment:
nitpick (non-blocking): A brief docstring describing what this method does would be nice.